sql/src/main/java/io/crate/exceptions/InvalidColumnNameException.java

```diff
@@ -21,16 +21,28 @@
 
 package io.crate.exceptions;
 
+import io.crate.metadata.TableIdent;
+
+import java.util.Collection;
+import java.util.Collections;
 import java.util.Locale;
 
-public class InvalidColumnNameException extends ValidationException {
+public class InvalidColumnNameException extends ValidationException implements TableScopeException {
```

Comment:
since both InvalidcolumnNameException and InvalidSchemaNameException are static validations i think those could be just unprotected, since ther eis no information shown about the table nor the schema